tmt/steps/report/__init__.py

```diff
@@ -4,6 +4,7 @@
 import click
 
 import tmt
+import tmt.result
```

Comment:
@happz this should be not needed, tmt exports `Result` in `__all__`

Reply:
line 85 can be changed to use directly `Result` I think

Reply:
This seems to be even unrelated change, I think I made it to make my IDE happy, but it's not touching any logging issue...

Reply:
Fixed in 39891de6dbf9fb4e01471bda84147df2156d9eea